### DISCUSSION TITLE
Why does your official site come up as malicious?

### DISCUSSION THREAD

**Discussion by 7ngel** [Belphemur/SoundSwitch#1390] [FAQ]
![image](https://github.com/Belphemur/SoundSwitch/assets/34433374/f6a31957-3fd8-447f-a9b4-2b5ce9ec28e0)

**Comment by Belphemur** [comment]
As you can see it only flag by 2 vendors. It's a false positive.

**Reply by 7ngel** [reply]
Just because it was flagged by 2 vendors doesn't automatically make it a false positive...

**Reply by Belphemur** [reply]
Just because it was flagged by two vendor doesn't make it a virus.

That goes both way. By the way, both vendor aren't reputable ones. Nonetheless, you do you :)
